### DISCUSSION TITLE
New release

### DISCUSSION THREAD

**Discussion by vladmandic** [vladmandic/sdnext#2587] [Announcements]
## Update for 2023-12-04

What's new? Native video in SD.Next via both **AnimateDiff** and **Stable-Video-Diffusion** - and including native MP4 encoding and smooth video outputs out-of-the-box, not just animated-GIFs.  
Also new is support for **SDXL-Turbo** as well as new **Kandinsky 3** models and cool latent correction via **HDR controls** for any *txt2img* workflows, best-of-class **SDXL model merge** using full ReBasin methods and further mobile UI optimizations.  

Full details: <https://github.com/vladmandic/automatic/discussions/99#discussioncomment-7754246>

**Comment by HonZuna** [comment]
SD.Next is amazing. Please improve its appearance; I am convinced that this is the main problem for many people. It does not look attractive or well-made at all, even though it actually is a great tool.

**Reply by vladmandic** [reply]
Can you be more specific? Which theme are you using, which areas you'd improve and how?

**Reply by Arctomachine** [reply]
There was information about possibility of adopting progress from UIUX fork. Is it still relevant and being considered?

**Reply by HonZuna** [reply]
I apologize, I am not a GUI expert, but the difference is really striking if you compare it to the GUIs of Auto1111, Focus, or InvokeAI. SD-Next looks outdated, almost as if it was hastily done. It lacks any kind of shading, and the elements are often misaligned. Something like a UIUX fork or the Material Design policy by Google would be great. 

I think it's one of the reasons why there are practically no tutorials on SD.Next online and why AI influencers and creators don't make videos in it. However, given how the development of Auto1111 is increasingly showing to be slower, it presents an opportunity for change.

**Reply by vladmandic** [reply]
i asked explicitly which theme you're using since different themes have a very different appearance.

**Reply by HonZuna** [reply]
I use the default, and the default is also shown in the illustrative images.

**Reply by Sunsphire** [reply]
The UX is better in SD.Next than in your examples. Fooocus and InvokeAI are more or less simplified UIs and that doesn't necessarily mean a "better UX" - you can achieve a similar result by customising your UI in SD.Next by having open just what you want. UI-UX looks more like a skin, I didn't try it but by design standards it looks more like a "retro style" than a real improvement (again, I didn't try it, just seen screenshots). But by all the design standards, SD.Next looks more modern than UI-UX with cleaner elements...maybe you feel like it looks dull but definitely not outdated or less well made. It looks more like a professional tool and less like a playful thing.

Also, definitely no shadings if you don't want an outdated look, Material Design you mentioned is exactly that, an improvement of the "flat design" trend - a lack of shadings in favour of flat colored, non shaded, elements. Skeuomorphic design is dead...let it rest.

SD.Next UI is fine...just needs a little more work with the accordion elements because it can be a little chaotic when they change position/state - maybe there is a way to keep all the options on top, under the Width/Height and having their state shown by the arrow.

**Comment by jjmenet** [comment]
Thank you for great update, I'm enjoying it.

It seems HDR parameters are not written to exif parameters or result page, right? Can they be included, for sake of reproducibility?

**Reply by vladmandic** [reply]
just added in dev, will be merged to master in the next release.
